packages/types/src/documents/app/row.ts

```diff
@@ -7,6 +7,7 @@ export enum FieldType {
   NUMBER = "number",
   BOOLEAN = "boolean",
   ARRAY = "array",
+  QUERY_ARRAY = "queryarray",
```

Comment:
One thing that we have been doing, in `packages/types/src/documents/app/table/schema.ts` - we've been defining the structure of Field schemas, the `QUERY_ARRAY` type - if it has any special properties etc it would be worth building out a specification for what the schema actually looks like/what is expected in the table/query schema definition. Right now these definitions are for tables, however someone could easily try to use this type for a table, so its probably best to define what is expected.